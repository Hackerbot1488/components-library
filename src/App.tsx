import React from "react";
import { Main } from "./layout/Main/Main";
import { Navigation } from "./layout/Navigation/Navigation";

export const App: React.FC = ({ children }) => {
	return (
		<>
			<Navigation key="navigation" />
			<Main key="main">{children}</Main>
		</>
	);
};
