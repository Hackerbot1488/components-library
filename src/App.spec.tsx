import { shallow } from "enzyme";
import React from "react";
import { App } from "./App";
describe("App component", () => {
	it("should render App component", () => {
		const component = shallow(<App />);
		expect(component).toMatchSnapshot();
	});
});
