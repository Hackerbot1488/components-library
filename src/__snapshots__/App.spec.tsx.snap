// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component should render App component 1`] = `
<div
  className="App"
>
  <Component
    key="navigation"
  />
  <Component
    key="main"
  />
</div>
`;
